src/app/common/transactions/bitcoin/utils.ts

```diff
@@ -9,6 +9,27 @@ import { satToBtc } from '@app/common/money/unit-conversion';
 
 import { BtcSizeFeeEstimator } from './fees/btc-size-fee-estimator';
 
+type BtcTxStatus = 'pending' | 'success';
+type BtcStatusColorMap = Record<BtcTxStatus, string>;
+
+const statusFromBitcoinTx = (tx: BitcoinTx): BtcTxStatus => {
+  if (tx.status.confirmed) return 'success';
+  return 'pending';
+};
+
```

Comment:
Nice work on this 👍 I was confused by the legacy `ColorsStringLiteral`. 

Maybe we can work with design on these colours to update them and have new semantic tokens for them? (We can make a task for it). 

- `pending` 
- `feedback-error` etc. 

I found some other hardcoded colours in `password-field.utils.ts` also